### DISCUSSION TITLE
Suggestion to collect the total size in use of the Azure Storage Account

### DISCUSSION THREAD

**Discussion by jihyungSong** [cloudforet-io/community#65] [Ideas]
**Propose Date** : 2022-12-06
**Proposer** : @jihyungSong 

---

Cloudforet is currently collecting Azure storage account data through [Azure Inventory Collector](https://github.com/cloudforet-io/plugin-azure-inven-collector).

Currently, only container data is being collected in the Storage Account, but it would be nice to collect the total number and total size of blobs stored in each container.

Both AWS S3 and Google Cloud's Cloud Storage, which are at the same level as Storage Account's Container, is collecting the total size and number of objects.

**Comment by ImMin5** [comment]
It would be helpful if user can check total size of blobs stored in each container.

You can track this issue [here](https://github.com/cloudforet-io/plugin-azure-inven-collector/issues/33).

and we are going to develop this feature asap!
